docs/book/src/developer/providers/webhooks.md

```diff
@@ -35,4 +57,4 @@ type Cluster struct {
 A detailed guide on the purpose of each of these tags is [here](https://book.kubebuilder.io/reference/markers/webhook.html).
 
 <!-- links -->
-[the Cluster webhook]: https://github.com/kubernetes-sigs/cluster-api/blob/release-1.1/internal/webhooks/cluster.go
+[the Cluster webhook]: https://github.com/kubernetes-sigs/cluster-api/blob/main/internal/webhooks/cluster.go
```

Comment:
```suggestion
[the Cluster webhook]: https://github.com/kubernetes-sigs/cluster-api/blob/release-1.1//internal/webhooks/cluster.go
```

cc @sbueringer

Reply:
Should this be 1.2?

Reply:
yes, thx

Reply:
Just missed that the current link points to release-1.1. Can someone open a follow-up PR (against main) to change that link to main as well?